datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -2377,6 +2377,17 @@ private void configureDataJobResolvers(final RuntimeWiring.Builder builder) {
                                   ? dataJob.getDataPlatformInstance().getUrn()
                                   : null;
                             }))
+                    .dataFetcher(
+                        "container",
+                        new LoadableTypeResolver<>(
+                            containerType,
+                            (env) -> {
+                              final DataJob dataJob = env.getSource();
+                              return dataJob.getDataPlatformInstance() != null
+                                  ? dataJob.getDataPlatformInstance().getUrn()
```

Comment:
looks like a copy paste error here - we want to have `dataJob.getContainer` here not `getDataPlatformInstance`

Reply:
nice catch! fixed in commit 21e00bc16ce54999501ce804758f0bd4b0b2931c